### PR TITLE
Create error pipeline processor

### PR DESCRIPTION
**Pull request by andrewkochura**

### CODE REVIEW

File: hyber/src/main/java/com/hyber/ErrorRespItemModel.java [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 class ErrorRespItemModel {
 
-    @SerializedName("code")
+    @SerializedName("mCode")
```
Comment (segabriel) [inline hyber/src/main/java/com/hyber/ErrorRespItemModel.java]: WTF?!?

File: hyber/src/main/java/com/hyber/ErrorRespItemModel.java [hunk 2]
```diff
@@ -6,9 +6,9 @@
...
     private Integer mCode;
-    @SerializedName("description")
+    @SerializedName("mDescription")
```
Comment (segabriel) [inline hyber/src/main/java/com/hyber/ErrorRespItemModel.java]: WTF?!? come back please!

File: hyber/src/main/java/com/hyber/ErrorRespItemModel.java (hunk 1 above)
Comment (andrewkochura) [inline hyber/src/main/java/com/hyber/ErrorRespItemModel.java]: done

File: hyber/src/main/java/com/hyber/ErrorRespItemModel.java (hunk 2 above)
Comment (andrewkochura) [inline hyber/src/main/java/com/hyber/ErrorRespItemModel.java]: done

**Pull request merged**